crates/fs/private/node.rs

```diff
@@ -45,9 +45,12 @@ pub enum PrivateNode {
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct PrivateRef {
-    pub(crate) saturated_name_hash: HashOutput, // Sha3-256 hash of saturated namefilter
-    pub(crate) content_key: ContentKey,         // A hash of ratchet key.
-    pub(crate) ratchet_key: RatchetKey,         // Encrypted ratchet key.
+    /// Sha3-256 hash of saturated namefilter.
+    pub(crate) saturated_name_hash: HashOutput,
+    /// Content key; Sha3-256 hash of the ratchet key.
+    pub(crate) content_key: ContentKey,
+    /// Ratchet key; namefilter-derived key.
```

Comment:
Typo, I think you meant this:
```suggestion
    /// Ratchet key; skip-ratchet-derived key.
```